modele-social/règles/profession-libérale.yaml

```
@@ -623,6 +625,129 @@ dirigeant . indépendant . PL . PAMC . IJSS:
+          plafond: 2.1875
+        - montant: 4 * cotisation de référence
+          plafond: 2.5625
+        - montant: 5 * cotisation de référence
+          plafond: 2.9375
+        - montant: 6 * cotisation de référence
+          plafond: 3.3125
+        - montant: 7 * cotisation de référence
+          plafond: 3.6875
+        - montant: 8 * cotisation de référence
+          plafond: 4.0625
+        - montant: 9 * cotisation de référence
+          plafond: 4.4375
+        - montant: 10 * cotisation de référence
+          plafond: 4.8125
+        - montant: 11 * cotisation de référence
+          plafond: 5.1875
+        - montant: 12 * cotisation de référence
+  références:
+    Site CAVP: https://www.cavp.fr/votre-profil/pharmacien-en-activite/vos-cotisations-regime-complementaire
+    Legifrance: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000039357705
+
+dirigeant . indépendant . PL . CAVP . invalidité et décès:
+  titre: invalidité et décès (CAVP)
+  remplace: cotisations et contributions . invalidité et décès
+  formule: 608 €/an
+  références:
+    Site CAVP: https://www.cavp.fr/votre-profil/pharmacien-en-activite/vos-cotisations
+
+dirigeant . indépendant . PL . CAVP . PCV:
+  titre: Prestation complémentaire de vieillesse (CAVP)
+  remplace: cotisations et contributions . PCV
+  applicable si: dirigeant . indépendant . PL . CAVP . directeur non médecin
+  formule:
+    somme:
+      - cotisations forfaitaire
+      - cotisations proportionnelle
+      - (- aide CPAM)
+    arrondi: oui
+  référence:
+    Taux 2021 (CAVP): https://www.cavp.fr/votre-profil/pharmacien-en-activite/vos-cotisations-regime-des-prestations-complementaires-vieillesse
+    La retraite en claire: https://www.la-retraite-en-clair.fr/parcours-professionnel-regimes-retraite/retraite-travailleurs-independants/retraite-complementaire-pharmaciens-cavp
+
+dirigeant . indépendant . PL . CAVP . PCV . cotisations forfaitaire:
+  formule: 1728 €/an
+
+dirigeant . indépendant . PL . CAVP . PCV . cotisations proportionnelle:
+  formule:
+    produit:
+      assiette: assiette des cotisations
+      taux: 0.30%
+      plafond: 5 * plafond sécurité sociale temps plein
+
+dirigeant . indépendant . PL . CAVP . PCV . aide CPAM:
+  somme:
+    - aide CPAM forfaitaire
+    - aide CPAM proportionnelle
+  référence:
+    CAVP: https://www.cavp.fr/votre-profil/pharmacien-en-activite/vos-cotisations-regime-des-prestations-complementaires-vieillesse
+    La retraite en claire: https://www.la-retraite-en-clair.fr/parcours-professionnel-regimes-retraite/retraite-travailleurs-independants/retraite-complementaire-pharmaciens-cavp
+
+dirigeant . indépendant . PL . CAVP . PCV . aide CPAM forfaitaire:
+  formule: 0.66 * dirigeant . indépendant . PL . CAVP . PCV . cotisations forfaitaire
+
+dirigeant . indépendant . PL . CAVP . PCV . aide CPAM proportionnelle:
+  formule: 0.5 * dirigeant . indépendant . PL . CAVP . PCV . cotisations proportionnelle
+
```

Top ! Pour plus de concision, tu aurais également pu écrire : 
```suggestion
dirigeant . indépendant . PL . CAVP . PCV . aide CPAM:
  somme:
    - 66% * cotisations forfaitaire
    - 50% * cotisations proportionnelle

  référence:
    CAVP: https://www.cavp.fr/votre-profil/pharmacien-en-activite/vos-cotisations-regime-des-prestations-complementaires-vieillesse
    La retraite en claire: https://www.la-retraite-en-clair.fr/parcours-professionnel-regimes-retraite/retraite-travailleurs-independants/retraite-complementaire-pharmaciens-cavp
```